about/introduction.rst

```diff
@@ -71,10 +71,10 @@ This documentation is organized into several sections:
 - **About** contains this introduction as well as
   information about the engine, its history, its licensing, authors, etc. It
   also contains the :ref:`doc_faq`.
-- **Getting Started** contains all necessary information on using the
-  engine to make games. It starts with the :ref:`Step by step
-  <toc-learn-step_by_step>` tutorial which should be the entry point for all
-  new users. **This is the best place to start if you're new!**
+- **Getting Started** contains all necessary information on using the engine to
+  make games. It starts with the :ref:`doc_getting_started_intro` sequence which
```

Comment:
```suggestion
  make games. It starts with the :ref:`doc_getting_started_intro` section which
```
minor wording nitpick, section sounds more correct than sequence for a written tutorial.